build/openrpc.json

```diff
@@ -0,0 +1,2613 @@
+{
```

Comment:
We should remove files that can be auto generated. We can add a CI task to build the spec into a non-main branch.

Reply:
You need this to be pre generated for the open RPC playground.

Reply:
Yes, I understand. It's generally standard though to do this as a CI task rather than require developers to do this on push. You can see how we do it for the eth1 spec here: https://github.com/quilt/eth1.0-specs/blob/pyspec/.github/workflows/gh-pages.yaml